.github/workflows/build_docker_image.yml

```diff
@@ -11,6 +11,8 @@ on:
 env:
   BUILDX_CACHE: /tmp/.buildx-cache
   CACHE_KEY: docker-erddap-buildx-
+  TOMCAT_AMD64_IMAGE: tomcat:10.1.26-jdk21-temurin-jammy@sha256:18952effb643bf192799e4ab2ca7c121d58871e96e5709fb5d405f4682a9aae7
+  TOMCAT_ARM64_IMAGE: tomcat:10.1.26-jdk21-temurin-jammy@sha256:4775c2227f16ee2726a35a1e97f43bfcb1a085cad23e169b1066ec9603826d8b
```

Comment:
@srstsavage should I revert this and let someone else submit this PR without a fork? OK is this OK for now?